library/validation/validate_core.php

```diff
@@ -71,6 +71,8 @@ function validateUsingPageRules($fileNamePath)
         print '<!---Start of page  form validation-->';
         print '<!--//include new rules of submitme functionallity-->';
         echo("\r\n");
+        //Not lbf forms use the new validation, please make sure you have the corresponding values in the list Page validation
+        $use_validate_js = 1;
         require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php");
         $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```

Comment:
Hi @sharonco-matrix ,

Do you have a link to the commit/branch fix on github?

thanks,
-brady